src/librbd/cache/LogMap.h

```diff
@@ -0,0 +1,391 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: move RWL classes to a `rwl` subdirectory of `librbd/cache` (i.e. match the namespace)

Reply:
Done